include/dlpack/dlpack.h

```diff
@@ -12,8 +12,15 @@
 #define DLPACK_EXTERN_C
 #endif
 
-/*! \brief The current version of dlpack */
-#define DLPACK_VERSION 050
```

Comment:
Note here, the leading `0` meant this was being stored as an octal value, so it was decimal `40`.